tests/basic_correctness/test_chunked_prefill.py

```diff
@@ -38,11 +39,16 @@ def test_models(
     chunked_prefill_token_size: int,
     enforce_eager: bool,
     tensor_parallel_size: int,
+    attention_backend: str,
+    monkeypatch,
 ) -> None:
     """
     Checks exact match decode between huggingface model and vllm runner with
     chunked prefill.
     """
+    monkeypatch.setenv("VLLM_ATTENTION_BACKEND", attention_backend)
+    assert os.getenv("VLLM_ATTENTION_BACKEND") == attention_backend
```

Comment:
For consistency, could you use `override_backend_env_variable`?

Reply:
Done!